lib/msal-browser/src/broker/nativeBroker/NativeMessageHandler.ts

```diff
@@ -113,6 +129,7 @@ export class NativeMessageHandler {
                 window.removeEventListener("message", this.windowListener, false);
                 this.messageChannel.port1.close();
                 this.messageChannel.port2.close();
+                this.handshakeEvent.endMeasurement({wamTimedOut: true, success: false});
```

Comment:
Can we call this "extensionHandshakeTimedOut" instead? There's actually no communication with the broker during handshake

Reply:
done.